lgc/patch/Patch.cpp

```diff
@@ -282,6 +282,7 @@ void Patch::addOptimizationPasses(legacy::PassManager &passMgr) {
     passMgr.add(createInstSimplifyLegacyPass());
     passMgr.add(createDivRemPairsPass());
     passMgr.add(createCFGSimplificationPass());
+    passMgr.add(createPatchReadfirstlaneOpt());
```

Comment:
Please move this to before the most recent createInstructionCombiningPass, and add a comment `// uses DivergenceAnalysis`.

The rationale is that in the future, with a more general `PatchReadfirstlaneOpt` pass, the lifting of readfirstlane can create new optimization opportunities for instruction combining.

Reply:
Done.